bin/cylc-suite-state

```diff
@@ -211,6 +227,21 @@ def main():
 
         options.cycle = str(my_target_point)
 
+    # Exit if both task state and message are to being polled
+    if options.status and options.msg:
+        sys.exit("ERROR: cannot poll both status and message")
+
+    if options.msg and not options.task and not options.cycle:
+        sys.exit("ERROR: need a taskname and cyclepoint")
+
+    # Exit if an invalid status is requested
+    if options.status:
+        if (options.status in TASK_STATUSES_ORDERED or
+                options.status in CylcSuiteDBChecker.STATE_ALIASES):
+            pass
+        else:
+            sys.exit("ERROR: invalid status '" + options.status + "'")
```

Comment:
Can become a single branch?

```python
if (option.status and
        option.status not in TASK_STATUSES_ORDERED and
        option.status not in CylcSuiteDBChecker.STATE_ALIASES):
    sys.exit("ERROR: invalid status '" + options.status + "'")
```

Alternatively, you can use `choices=LIST` in `parser.add_option` instead of do this checking yourself.